src/main/java/com/microsoft/spring/data/gremlin/exception/GremlinUpdationException.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.spring.data.gremlin.exception;
+
+import org.springframework.dao.DataAccessResourceFailureException;
+
+public class GremlinUpdationException extends DataAccessResourceFailureException {
```

Comment:
Why it's named `Updation` but not `Update`?

Reply:
following the previous naming rule insert to insertion.